lib/api/decorators/schema.rb

```diff
@@ -48,8 +48,8 @@ def schema(property,
                      name = call_or_translate(name_source)
                      schema = ::API::Decorators::PropertySchemaRepresenter.new(type: type,
                                                                                name: name,
-                                                                               required: required,
-                                                                               writable: writable)
+                                                                               required: call_or_use(required),
```

Comment:
I hate that this line got this long, but it is basically only this darn namespace prefix...

I'd rather keep it as long if there are no objections...